src/olympia/promoted/views.py

```diff
@@ -0,0 +1,35 @@
+import stripe
+
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+
+import olympia.core.logger
+
+from olympia.amo.decorators import post_required
+
+from .utils import create_stripe_webhook_event
+
+
+log = olympia.core.logger.getLogger("z.promoted")
+
+
+@post_required
+@csrf_exempt
+def stripe_webhook(request):
+    payload = request.body
```

Comment:
looks like the in stripe docs the payload is parsed as JSON before passing it to `construct_event`.

Reply:
not for signed requests